docs/self-service-account-api-descriptor.yml

```diff
@@ -339,6 +339,40 @@ paths:
             $ref: '#/definitions/EnrolledWebAuthnCredential'
         '401':
           description: Invalid Token
+  /api/consent:
+    get:
+      tags:
+        - Self Account
+      summary: Get list of consent for the current user
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: The user consent list
+          schema:
+            $ref: '#/definitions/UserConsentList'
+        '401':
+          description: Invalid Token
+  /api/consent/{consentId}:
+    get:
```

Comment:
Don't see the interest to have a GET method on a specific consent here as the listContent already provide all the information.